apps/frontend/src/pages/route-details/libs/helpers/get-google-maps-url.helper.ts

```diff
@@ -0,0 +1,9 @@
+import { configureString } from "~/libs/helpers/helpers.js";
+
+const getGoogleMapsUrl = (lat: number, lng: number): string =>
```

Comment:
let's rename it for clarity, don't forget about name
```suggestion
const getGoogleMapsPointUrl = (lat: number, lng: number): string =>
```